app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
-    # Write your code here
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int | float) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
```

Comment:
```suggestion
        return Distance(self.km + other)
```
Due to type annotations it is always true

Reply:
Fix the same in other places